src/styles/_catalog.scss

```diff
@@ -9,14 +11,32 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 488px) {
```

Comment:
minor: it is not necessary to write this media query. If other media queries conditions are false the default one will be applied